oleesir/Banking-app
# #164722785Users should be able to see the login page

#### What does this PR do?
implement login-template
#### Description of Task to be completed?
implement login-template
#### How should this be manually tested?
* Navigate to the UI folder and click on login.html in your browser
* You will see a login template

#### What are the relevant pivotal tracker stories?
[#164722785](https://www.pivotaltracker.com/story/show/164722785)
#### Any background context you want to add?
N/A
#### Screenshots
<img width="1280" alt="Screenshot 2019-03-20 at 7 35 43 PM" src="https://user-images.githubusercontent.com/17679511/54712362-5f4d2600-4b4c-11e9-8567-7be2ca175b74.png">
